tests/func/test_add.py

```diff
@@ -292,6 +293,11 @@ def test_add_filtered_files_in_dir(
         assert stage.outs[0].def_path in expected_def_paths
 
 
+def test_add_glob_no_result_exception(dvc):
+    with pytest.raises(GlobDoesNotMatchError):
+        dvc.add(["invalid*"], glob=True)
+
+
 class TestAddExternal(TestAdd):
     @pytest.fixture
```

Comment:
Let's just add one unit test for `glob_targets` and not introduce add/pull/etc func tests, please.